logging/log.go

```diff
@@ -62,5 +56,5 @@ func Init(o Options) {
 	}
 }
 
-func ApplicationLog() *logrus.Logger { return appLog }
+func ApplicationLog() *logrus.Logger { return logrus.StandardLogger() }
```

Comment:
This one is, basically, available through `logrus.XXX` or, with the alias, `log.XXX`. Do you think it's worth it to export such function?